sed/calibrator/energy.py

```diff
@@ -95,6 +96,7 @@ def __init__(
         self.calibration: Dict[Any, Any] = {}
 
         self.tof_column = self._config["dataframe"]["tof_column"]
+        self.tof_ns_column = self._config["dataframe"].get("tof_ns_column", None)
```

Comment:
I don't find this a good way to initialize these parameters. It forces us to either initialize everything, with defaults for every possible option, or to use a different syntax for non-default parameters. I would raher not initialize anything and check at each function call if the value is there. This could be wrapped in a better config manager class, which would have all error handling etc...

For the time being, I will add the more general ones to the default, but keep this structure for the more specific values.